kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -79,17 +77,14 @@ def get_kubeconfig_path():
 
 def get_label():
     test = os.environ.get("PYTEST_CURRENT_TEST")
+    test = test.split(" ")[0]  # if invoked from setup, will have suffix ' (setup)', e.g.
     label = "".join(filter(str.isalnum, test)).lower()
-    return label[-63]
+    return label[-63:]
```

Comment:
... as a result, looking at just this one character was unreliable. here we also make it longer so it's more meaningful.